### DISCUSSION TITLE
automatically join the chat when a streamer is live

### DISCUSSION THREAD

**Discussion by maxcamorra** [Chatterino/chatterino2#3858] [Ideas]
It would be nice to automatically connect to the chat when the streamer is live and automatically leave the chat when he goes offline.

**Comment by leon-richardt** [comment]
Could you describe in a little more detail what use case this would address?

**Reply by maxcamorra** [reply]
because i like to be in multiple chats and often I don't see the notifications because playing something or being afk so in this case chatterino would automatically add the streamer's chat that went live to my tab split so I will never forget to join his chat.

**Reply by leon-richardt** [reply]
That part seems sensible. Why would you like to automatically disconnect once the streamer goes offline?
